hammers/query.py

```diff
@@ -201,6 +201,67 @@ def future_reservations(db):
     '''
     return db.query(sql, limit=None)
 
+@query
+def active_instances(db):
+    '''
+    Get active instances.
+    '''
+    sql = '''
+    SELECT DISTINCT uuid, user_id, project_id 
+    FROM nova.instances 
+    WHERE deleted_at is NULL;
+    '''
+    return db.query(sql, limit=None)
+
+@query
+def orphans(db, orphan_type):
+    '''
+    Get orphans of certain type that haven't been deleted,  along with users' and projects' information. 
+    '''
+    db_tables = []
+    conditions = '1'
```

Comment:
What is this line for?

Reply:
initialize the condition variable. if no condition overwrites later, it will run query like "select * from <table> where 1", which is the same as selecting everything without condition.

Reply:
My understanding of the code is that if orphan_type is not 'lease' nor 'instance', it will raise an exception.

I like the idea of initialising the conditions variable, but I would set it to '0' so that nothing is detected in case a code change incorrectly skips the exception.